tests/test_verify.py

```diff
@@ -0,0 +1,18 @@
+
+##Write a software that verifies if a number is present in a pre-defined array.
+##Output example:
+##Insert number 3
+##The number 3 is [not] present in the array.
+##
+test_array = [2,4,6,8,10,12,14,16,18,20]
```

Comment:
potresti rinominare questo i success_test_array in modo da crearne un altro chiamato fail_test_array in cui inserisci i valori 35 e 55 che usi alla riga 16 per rendere più chiaro cosa sia la lista su cui stai iterando

Reply:
> potresti rinominare questo i success_test_array in modo da crearne un altro chiamato fail_test_array in cui inserisci i valori 35 e 55 che usi alla riga 16 per rendere più chiaro cosa sia la lista su cui stai iterando

done